contribs/gmf/examples/editfeatureselector.js

```diff
@@ -120,14 +130,36 @@ app.MainController = function($scope, gmfThemes, gmfUser, ngeoToolActivateMgr) {
   });
 
   /**
+   * @type {Object.<ol.geom.GeometryType, Array.<ol.style.Style>>}
+   * @private
+   */
+  this.editingStyles_ = ngeo.style.createDefaultEditingStyles();
+
+  /**
    * @type {ol.layer.Vector}
    * @export
    */
   this.vectorLayer = new ol.layer.Vector({
     source: new ol.source.Vector({
       wrapX: false,
       features: new ol.Collection()
-    })
+    }),
+    style: function(feature, resolution) {
+      // (1) Use the geometry collection style, which works fine for any
+      //     geometry types
+      var styles = [].concat(this.editingStyles_[
+        /** @type {ol.geom.GeometryType<string>} */ ('GeometryCollection')
+      ]);
+
+      // (2) Then, add the vertex style if the geometry is different than points
+      var geom = feature.getGeometry();
+      console.assert(geom);
+      var type = geom.getType();
+      if (type !== 'Point') {
+        styles.push(ngeoFeatureHelper.getVertexStyle(true));
+      }
+      return styles;
+    }.bind(this)
```

Comment:
Well, for the vertex it's nice that we already have a function that takes care of it for us, i.e. `ngeoFeatureHelper.getVertexStyle(true)`.  It would be more complex to define the style ourself in the example.